csharp/ql/lib/change-notes/2022-03-03-recursive-qltest-extraction.md

```diff
@@ -0,0 +1,4 @@
+---
+category: breaking
+---
+* `codeql test run` now extracts source code recursivelly from sub folders. This may break existing tests that have other tests in nested sub folders, as those will now get the nested test code included.
```

Comment:
Alternatively:
```suggestion
* `codeql test run` now extracts source code recursively from sub folders. This may break existing tests that have other tests in nested sub folders, as those will now get the nested test code included.
```